### ISSUE TITLE
Does not build in Xcode 10 by dragging project into workspace

### ISSUE THREAD

**Issue by drkibitz** [ReSwift/ReSwift-Router#104]
The problem is the iOS target's *copy-frameworks* phase. This fails, because I am not using carthage in my build pipeline for my application, purposefully.

I don't think a framework should have a "copy-frameworks" phase. This should be reserved for executables.

**Comment by drkibitz** [comment]
This is a false alarm, seems I had an invalid submodule propagated through carthage in the first place, and ReSwift-Router's copy-frameworks phase was what was failing. Not sure how it got in this state, but a fresh clone builds. Sorry for the false alarm.

@DivineDominion Though if you would like to continue the discussion about why the "copy-frameworks" phase is needed, that's fine. Because I still do fine that odd.

**Comment by mjarvis** [comment]
Looks like I made a mistake back when I reorganized things. Thanks for pointing this out!

**Comment by drkibitz** [comment]
Sounds good, thanks @mjarvis !

**Comment by drkibitz** [comment]
@mjarvis I just remembered a theory about why my Git/Carthage barfed. So the repo is `ReSwift/ReSwift-Router`, but the Carthage instructions say `github "ReSwift/ReSwiftRouter"`. Test Github serves up a 301 at this URL, but there might be something weird going on with Carthage in terms of parsing and what not. I remember I changed my Cartfile to ``github "ReSwift/ReSwift-Router"` and recloned and everything was working fine from that point on. I thought it might have been a fluke, but maybe it is related to that `-` and the 301?
